src/app/[entity]/[repo]/issues/new/page.tsx

```diff
@@ -1,7 +1,284 @@
+import { TextArea } from '@/components/ui/textarea';
+import {
+  DropdownMenu,
+  DropdownMenuTrigger,
+  DropdownMenuContent,
+  DropdownMenuLabel,
+  DropdownMenuSeparator,
+  DropdownMenuGroup,
+  DropdownMenuItem
+} from '@/components/ui/dropdown-menu';
+import Link from 'next/link';
+
 export default function RepoIssueNewPage() {
+  const router = useRouter();
+  const titleRef = useRef<HTMLInputElement>(null);
+  const commentRef = useRef<HTMLTextAreaElement>(null);
+  const assigneesFilterRef = useRef<HTMLInputElement>(null);
+  const labelsFilterRef = useRef<HTMLInputElement>(null);
+  const [errorMsg, setErrorMsg] = useState<string>('');
+  const [selectedLabels, setSelectedLabels] = useState<string[]>([]);
+  const { picture, initials, isLoggedIn } = useSession();
+
+  const handleSubmit = useCallback(
+    (e: React.FormEvent<HTMLFormElement>) => {
+      e.preventDefault();
+
+      // todo: check the correct localstorage key for nsec
+      let privateKey = localStorage.getItem('nostr:privkey')
+
+      if (!isLoggedIn || !privateKey) {
+        setErrorMsg('Sign in with your private key to create issues.')
+        setTimeout(() => {
+          setErrorMsg('')
+        }, 6000)
+        return
+      }
+
+      let event = {
+        kind: 1,
+        created_at: Math.floor(Date.now() / 1000),
+        tags: [],
+        content: `${titleRef.current?.value}\n\n${commentRef.current?.value}` || "",
+        pubkey: getPublicKey(privateKey),
+        id: "",
+        sig: ""
+      }
+
+      event.id = getEventHash(event)
+      event.sig = signEvent(event, privateKey)
+
+      let ok = validateEvent(event)
+      let veryOk = verifySignature(event)
+
+      // todo: publlish to defaultRelays with NostrContext
+      console.log('Event created but not published: ', event.id)
+
+      // todo: route to issues page of the correct repo
+      router.push("/issues");
+    },
+    [router]
+  );
+
+  const selectLabel = (label: string) => {
+    if (!selectedLabels.includes(label)) {
+      setSelectedLabels([
+        label,
+        ...selectedLabels
+      ]);
+    } else {
+      setSelectedLabels(selectedLabels.filter(l => l !== label))
+    }
+  }
+
   return (
-    <>
-      <h1>New Issue Page</h1>
-    </>
+    <div className='flex justify-center mt-8 gap-2'>
+      <Avatar className="w-8 h-8">
+        <AvatarImage src={picture} />
+        <AvatarFallback>{initials}</AvatarFallback>
+      </Avatar>
+      <div>
+        <div className="sm:mx-auto sm:w-full sm:max-w-md">
+          <div className="bg-[#171B21] py-2 px-4 shadow sm:rounded-lg sm:px-10">
+            <form className="space-y-6" onSubmit={handleSubmit}>
+              <div>
+                <div className="mt-2">
+                  <Input
+                    id="title"
+                    name="title"
+                    type="text"
+                    required
+                    placeholder="Title"
+                    className="w-full block w-96"

```

Comment:
```suggestion
                    className="w-full block"
```
only one is needed.  fixed widths like `w-96` make it difficult for responsiviness

Reply:
Good catch. I had problems getting the page to hot-reload because of the hydration issues so I must have been desperate to see some width change so I added two. :smile: